scripts/diff_build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+if [ ! -z "$1" ]; then 
+  CC="$1"
```

Comment:
This does not work: because `CC` is not `export`ed, `make` won't see it and always build with the default compiler. This just wasn't noticed because the call to `$CC --version` below still sees the variable and calls the correct compiler.

I suggest to set the variable in the CI job definition and not in the script.

Reply:
done.